<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>realizedServiceInfo</class>
 <widget class="QDialog" name="realizedServiceInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="line_ShortDescription">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>539</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>70</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="placeholderText">
    <string>Descrição curta do serviço</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>410</y>
     <width>175</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="lbl_InsertWorkCost">
      <property name="text">
       <string>Insira as Horas Trabalhadas</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lbl_PartsCost">
      <property name="text">
       <string>Custo das Peças </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lbl_totlaServiceCost">
      <property name="text">
       <string>Custo Total </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTextEdit" name="txt_FullDescription">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>541</width>
     <height>321</height>
    </rect>
   </property>
   <property name="autoFormatting">
    <set>QTextEdit::AutoNone</set>
   </property>
   <property name="tabChangesFocus">
    <bool>true</bool>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Add_PartsUsedInTheService">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>181</width>
     <height>34</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Listar usadas nesse serviço</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_WhichCarService">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>480</y>
     <width>59</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>410</y>
     <width>81</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QDoubleSpinBox" name="SpinHandWorkCost">
      <property name="singleStep">
       <double>5.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QDoubleSpinBox" name="Spin_PartsCost">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="singleStep">
       <double>5.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QDoubleSpinBox" name="Spin_TotalserviceCost">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="singleStep">
       <double>5.000000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>141</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Serviço realizado em:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_date">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>221</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>line_ShortDescription</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>txt_FullDescription</zorder>
  <zorder>btn_Add_PartsUsedInTheService</zorder>
  <zorder>lbl_WhichCarService</zorder>
  <zorder>label</zorder>
  <zorder>lbl_date</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
